app/src/main/java/ch/epfl/culturequest/ui/scan/ScanFragment.java

```diff
@@ -26,11 +33,19 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         binding = FragmentScanBinding.inflate(inflater, container, false);
         View root = binding.getRoot();
 
+        // Creates the LocalStorage to store the images locally
+        ContentResolver resolver = requireActivity().getApplicationContext().getContentResolver();
+        localStorage = new LocalStorage(resolver);
+
         // Adds a listener to the scan button and performs action
-        binding.scanAction.scanButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                //TODO: Add the action to perform when the button is clicked
+        binding.scanAction.scanButton.setOnClickListener(view -> {
+            // Creates the bitmap image from the drawable folder
+            Bitmap bitmap = BitmapFactory.decodeResource(getResources(), R.drawable.joconde);
+            boolean isWifiAvailable = false;
```

Comment:
Why do you set isWifiAvailable to false? Is it because you don't want it to call googleLensAPI?

Reply:
As we don't have access to the API for now I just put it false by default ;)